script/generatecert

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+set -x
+
+mkdir cert
+chmod 700 cert
+# Generate private key with password "password"
+openssl genrsa -aes128 -passout pass:password -out cert/server.pem 2048
```

Comment:
Hmm, didn't seem to work...

```
$ openssl genrsa -aes128 -out cert/server.pem 2048
Generating RSA private key, 2048 bit long modulus
...........+++
.......+++
e is 65537 (0x10001)
Enter pass phrase for cert/server.pem:
aborted!
5450:error:0906906F:PEM routines:PEM_ASN1_write_bio:read key:/BuildRoot/Library/Caches/com.apple.xbs/Sources/OpenSSL098/OpenSSL098-59/src/crypto/pem/pem_lib.c:334:
$ openssl genrsa -aes128 -passout pass: -out cert/server.pem 2048
Generating RSA private key, 2048 bit long modulus
..+++
.....................+++
e is 65537 (0x10001)
5481:error:0906906F:PEM routines:PEM_ASN1_write_bio:read key:/BuildRoot/Library/Caches/com.apple.xbs/Sources/OpenSSL098/OpenSSL098-59/src/crypto/pem/pem_lib.c:334:
```

Was looking at some [StackOverflow suggestions](http://serverfault.com/questions/366372/is-it-possible-to-generate-rsa-key-without-pass-phrase), but didn't want to mess with this script—trying to avoid breaking anything!